.env.example

```diff
@@ -48,9 +51,11 @@ ETHERSCAN_API_KEY=
 # These need to be deployed and set manually, ideally from circle's stablecoin-evm repo
 
 # Mainnets
+USDC=
```

Comment:
Why not `USDC_ETHEREUM_IMPLEMENTATION`?

Reply:
makes sense

Reply:
But I think is better with `MAINNET` instead of `ETHEREUM`